dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/UsersServiceImpl.java

```diff
@@ -514,6 +518,8 @@ public int getUserIdByName(String name) {
             }
         }
 
+        accessTokenMapper.deleteAccessTokenByUserId(id);
```

Comment:
We may need to add transaction here, this service method involves 2 DML operations.

Reply:
@ruanwenjun  Thank you for your reminder, updated.